pm/sender_test.go

```diff
@@ -91,6 +91,49 @@ func TestSenderEV(t *testing.T) {
 	assert.Zero(ticketEV(ticketParams.FaceValue, ticketParams.WinProb).Cmp(ev))
 }
 
+func TestValidateSender(t *testing.T) {
```

Comment:
Think this is triggering a CI error since this func name is already declared https://github.com/livepeer/go-livepeer/blob/master/pm/sendermonitor_test.go#L473

Reply:
indeed, fixed here 1cec746